CHANGELOG.md

```diff
@@ -40,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecations
 - The role of `register_custom_architecture` has been taken over by
   `baybe.surrogates.base.SurrogateProtocol`
+- `BayesianRecommender.surrogate_model` has been replaced with `get_surrogate`
```

Comment:
It seems like the serialization example might be unhappy since `examples/Serialization/create_from_config` is responsible for the docs currently failing. Can you quickly double check?

Reply:
Thanks for highlighting. This is in fact an ugly one and I can only fix it using a workaround for now. That issue stems back to a problem that is on my radar since long and also blocks us at some other points – basically it's a bad design in our serialization machinery where we add the tag of the subclass. The good news: with the new cattrs release that came out lately, there is now also official support for the feature needed to resolve the problem. However, I won't manage to do so before my vacation. Need to sit down for a day or two when I'm back and finally remove this blocker in one dedicated PR. 

Hope the two of you can live with the workaround 🙃

Reply:
No complaints so far, hence I'll close